src/main.js

```diff
@@ -22,6 +24,16 @@ var game = new Phaser.Game(config);
 
 function preload ()
 {
+    this.load.spritesheet({
+        key: 'player',
+        url: playerPNG,
+        frameConfig: {
+            frameWidth: 32,
+            frameHeight: 32,
```

Comment:
The size of the images in [the sprite](https://github.com/VictorCabello/MoreComplexGame/blob/e53486777b12646932f4fbaea5efd3b86a76308e/src/assets/img/king.png) is 32 px by 32px